eclipse/ide-target-platform/ide-target-platform.target

```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<?pde version="3.8"?><target name="GCP Development Target (ide-target-repository)" sequenceNumber="18">
-<locations>
-<location path="${project_loc:/ide-target-platform.repo}/target/repository/plugins" type="Directory"/>
```

Comment:
…so we can't just use a common copy like we did with the `ide-target-platform.target`